src/misc/filters.js

```diff
@@ -1,9 +1,13 @@
-goog.provide('ngeo.filters');
+goog.provide('ngeo.misc.filters');
 
 goog.require('ngeo');
 goog.require('ol.math');
 goog.require('ol.string');
 
+ngeo.misc.filters = angular.module('ngeoAngularFilters', []);
+
+ngeo.module.requires.push(ngeo.misc.filters.name);
```

Comment:
You add it here, and remove it later ?

Reply:
:-)

Reply:
Yes, I remove these ngeo.module.requires.push in the other PR.

I will rebase and remove lines later in #3455

Reply:
OK